src/views/js/utilities.js

```diff
@@ -0,0 +1,147 @@
+/**
+ * @author colsen@sfy.co.za
+ * @since 14-Jul-17.
+ */
+
+function getStringObjectFromCookie(objectKey) {
+    var allKeysAndValues = document.cookie.split(";");
+
+    for (var i = 0; i < allKeysAndValues.length; i++) {
+        var keyAndValue = allKeysAndValues[i];
+        var keyAndValueAsArray = keyAndValue.split("=");
+        var keyStr = keyAndValueAsArray[0].trim();
+
+        if (keyStr === objectKey) {
+            return keyAndValueAsArray[1];
+        }
+    }
+    setStringObjectOnCookie(objectKey, "");
+    return getStringObjectFromCookie(objectKey);
+}
+
+function setStringObjectOnCookie(objectKey, objectValue) {
+    var allKeysAndValues = document.cookie.split(";");
+
+    if (!(allKeysAndValues > 1)) {
+        newValue = objectKey + "=" + objectValue;
+        document.cookie = newValue;
+        return;
+    }
+
+    for (var i = 0; i < allKeysAndValues.length; i++) {
+        var keyAndValue = allKeysAndValues[i];
+        var keyAndValueAsArray = keyAndValue.split("=");
+
+        var keyStr = keyAndValueAsArray[0].trim();
+        if (keyStr === objectKey) {
+            var newValue = objectKey + "=" + objectValue;
+            document.cookie = document.cookie.replace(keyAndValue, newValue);
+            return;
+        }
+    }
+
+    newValue = objectKey + "=" + objectValue;
+    document.cookie = document.cookie + ";" + newValue;
+}
+
+function readSettings() {
+    var cookies = document.cookie.split(";");
+
+    for (var i = 0; i < cookies.length; i++) {
+        var cookieValue = cookies[i].split("=");
+        var name = cookieValue[0].trim();
+
+        if (name == "color") {
+            var color = cookieValue[1];
```

Comment:
Rather create a function that does the disabling of all themes and then only activate the theme you need. This will shorten this method by quite a few lines